tests/page-objects/default/enketo/pregnancy.wdio.page.js

```diff
@@ -0,0 +1,210 @@
+const deliveryDate = () => $(`${FORM} section[name="/pregnancy/gestational_age/method_edd"] input.ignore.input-small`);
+const ancVisitsPast = () => $(`${FORM} input[name="/pregnancy/anc_visits_hf/anc_visits_hf_past/visited_hf_count"]`);
+const eddConfirmation = () => $(`${FORM} ` +
+  `span[data-itext-id="/pregnancy/gestational_age/method_lmp_summary/u_edd_note:label"] ` +
+  `span[data-value=" /pregnancy/gestational_age/g_edd "]`);
+const weeksPregnantConfirmation = () => $(`${FORM} ` +
+  `span[data-itext-id="/pregnancy/gestational_age/method_lmp_summary/lmp_note:label"] ` +
+  `span[data-value=" /pregnancy/weeks_since_lmp_rounded "] `);
+const futureVisitDate = () => $(`${FORM} ` +
+  `section[name="/pregnancy/anc_visits_hf/anc_visits_hf_next/anc_next_visit_date"] input.ignore.input-small`);
+const riskFactors = (value) => $$(`${FORM} ` +
+  `input[name="/pregnancy/risk_factors/risk_factors_present/${value}_condition"]`);
+const patientNameSummary = () => $(`${SUMMARY_SECTION} span[data-value=" /pregnancy/patient_name "]`);
+const weeksPregnantSummary = () => $(`${SUMMARY_SECTION} span[data-value=" /pregnancy/weeks_since_lmp_rounded "]`);
+const eddSummary = () => $(`${SUMMARY_SECTION} span[data-value=" /pregnancy/summary/edd_summary "]`);
+const riskFactorsSummary = () => $$(`${SUMMARY_SECTION} ` +
+  `:not(label.disabled):not(label.or-appearance-h3) span[data-itext-id*="/pregnancy/summary/r_risk"]`);
+const dangerSignsSummary = () => $$(`${SUMMARY_SECTION} ` +
+  `:not(label.disabled) span[data-itext-id*="/pregnancy/summary/r_danger_sign_"]`);
+
+const selectGestationAge = async (value = GESTATION_AGE.edd) => {
+  const getAge = await gestationalAge(value);
+  await getAge.waitForDisplayed();
+  await getAge.click();
+};
+
+const setDeliveryDate = async (value = moment().add(1, 'month').format('YYYY-MM-DD')) => {
+  const date = await deliveryDate();
+  await date.waitForDisplayed();
+  await date.setValue(value);
+};
+
+const getConfirmationDetails = async () => {
+  return {
+    eddConfirm: await eddConfirmation().getText(),
+    weeksPregnantConfirm: await weeksPregnantConfirmation().getText(),
+  };
+};
+
+const setANCVisitsPast = async (value = 0) => {
+  const visits = await ancVisitsPast();
+  await visits.waitForDisplayed();
+  await visits.setValue(value);
+};
+
+const setFutureVisitDate = async (value = moment().add(1, 'day').format('YYYY-MM-DD')) => {
+  const date = await futureVisitDate();
+  await date.waitForDisplayed();
+  await date.setValue(value);
+};
+
+const selectYesNoOption = async (selector, value = 'yes') => {
+  const element = await $(`${FORM} ${selector}[value="${value}"]`);
+  await element.waitForDisplayed();
+  await element.click();
+  return value === 'yes';
+};
+
+// The selector for the 'riskFactors' is different depending on the option selected in the 'first pregnancy' question
+const selectAllRiskFactors = async (value = FIRST_PREGNANCY_VALUE.no) => {
+  const cbRisks = await riskFactors(value);
```

Comment:
What does cb stand for?

Reply:
It was just for comboBox, just to differentiate the variable's name 😅